setup.py

```diff
@@ -32,7 +32,7 @@ def find_version(*file_paths):
 install_requires = [
     'cached-property >= 1.2.0, < 2',
     'docopt >= 0.6.1, < 1',
-    'PyYAML >= 3.10, < 5',
+    'PyYAML >= 3.10, < 5.3',
```

Comment:
do we suspect that pyYAML will make breaking changes in v5.3?

```suggestion
    'PyYAML >= 3.10, < 6',
```

Reply:
I just wanted to be more conservative there. I think it would be better to review PyYAML's changelog before the next version update in setup.py.

Reply:
Everything else is pinned at a major version

Reply:
fair enough, +1 for consistency :)